openapi/external.yaml.template

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.1
 info:
-  version: 1.0.0
+  version: 0.0.0 
```

Comment:
the version in template differs from generated one

Reply:
If I understood what you're saying, this is intended. the `bundle-apis-pec` command does take the version from `package.json`, putting it in the final YAML generated. We also edited the `version` script to automate the specs bundling in the deploy pipeline.